v2/cmd/wails/internal/commands/dev/dev.go

```diff
@@ -473,19 +467,25 @@ func runFrontendDevWatcherCommand(cwd string, devCommand string, discoverViteSer
 	LogGreen("Running frontend DevWatcher command: '%s'", devCommand)
 	var wg sync.WaitGroup
 	wg.Add(1)
+	var state int32
 	go func() {
 		if err := cmd.Wait(); err != nil {
-			if err.Error() != "exit status 1" {
+			wasNotCanceled := atomic.CompareAndSwapInt32(&state, 0, 2)
```

Comment:
Was wondering if we should define consts for the 0,1 and 2 values for readability?

Reply:
Yeah, absolutely 👍